src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -1097,6 +1097,11 @@ namespace OpenLoco::Gfx
         return (1 << 29) | (colour << 19) | image;
     }
 
+    uint32_t recolour2(uint32_t image, uint8_t colour1, uint8_t colour2)
+    {
+        return (1 << 29) | (1 << 31) | (colour1 << 19) | (colour2 << 24) | image;
```

Comment:
I think this kind of expression is more readable in bit order. What about this?
```suggestion
        return (1 << 31) | (1 << 29) | (colour2 << 24) | (colour1 << 19) | image;
```

Reply:
In one of the other PRs I've labelled these magic constants which is even better